### PR TITLE
fix: downgrade the pydata sphinx theme

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -26,7 +26,7 @@ classifiers = [
 ]
 dependencies = [
     "Sphinx>=4.2.0",
-    "pydata-sphinx-theme>=0.14.0",
+    "pydata-sphinx-theme==0.14.4",
```
Comment (clatapie) [inline pyproject.toml:29]: Should we be less restrictive by asking `<=0.14.4`?

Comment (RobPasMue) [reply pyproject.toml:29]: I'd agree here... pinning down a specific PyData Sphinx Theme version seems too much for me

Comment (RobPasMue) [inline pyproject.toml:29]:
```suggestion
    "pydata-sphinx-theme>=0.14,<0.15",
```

Something like this seems better suited

Comment (Revathyvenugopal162) [reply pyproject.toml]: if we are not giving the specified version, dependabot will not open a PR right? but its ok, we can keep an eye on their release

Comment (RobPasMue) [reply pyproject.toml]: Dependabot should open a PR on upper limits as well (if I remember properly)

**Review by RobPasMue — APPROVED** [review]

**Pull request merged**